README.md

```diff
@@ -24,6 +24,22 @@ dataset/aptos/
 
 ## Parameters
 
+`num_images`: Gives total number of images. 70:10:20 split implemented automatically for train:val:test. To change the train:val:test ratio, make the change in `APTOSDataset` class. Minimum 1000 images are set to class `DataProcessor`. If `num_images` > 1000, it will automatically set that as number of images used.
+
+`train_batch_size`: Currently set to 25 as the machine can handle only that, however higher batch_size ~ 32 is recommended.
+
+`valid_batch_size`: Currently set to 25 as the machine can handle only that, however higher batch_size ~ 32 is recommended.
+
+`test_batch_size`: Currently set to 2 as the inference can happen only for that, however higher batch_size ~ 25 is recommended.
+
+`timestep`: Currently experimented with 500, 80, 60, 50. Optimally ~ 60 is recommended.
+
+`num_classes`: Selected as 5 for 5 classes - 0, 1, 2, 3, 4. Don't change it unless you change the dataset.
+
+`include_guidance`: Ensures that DCG priors are used in diffusion. `true` indicates using DCG priors for diffusion, `false` indicates not using DCG priors for diffusion.
+
+`weight`: For a detailed description look at Diffusion section of README.
```

Comment:
Can you add a link to the diffusion section instead of just mentioning it?

Reply:
Added, if you think it looks good, can you resolve the comment?